game.go

```diff
@@ -99,9 +103,8 @@ mainloop:
 		default:
 			g.screen.Tick(Event{Type: EventNone})
 		}
-		update := time.Now()
-		g.screen.delta = update.Sub(clock).Seconds()
-		clock = update
+
 		g.screen.Draw()
+		time.Sleep(time.Duration((update.Sub(time.Now()).Seconds()*1000.0)+1000.0/g.screen.fps) * time.Millisecond)
```

Comment:
@JoelOtter No, it doesn't, because ```g.screen.fps``` is defined as a float64.  It's a bit of a hack, but "division by zero" is handled as infinity in floating point arithmetic rules, which when passed to ```time.Duration``` results in a very small negative number, and sleep returns instantly when given a negative number.  So, never defining ```g.screen.fps``` is the same as not slowing down the mainloop at all.
I can add a comment about this.